lib/compress.js

```diff
@@ -3485,7 +3496,7 @@ merge(Compressor.prototype, {
                                 operator: car.operator,
                                 expression: left
                             });
-                        }
+                        } else car.write_only = false;
```

Comment:
When doing maintenance in the future, could you please make a proper indented else `{` block `}` here?